internal/driver/readhandler.go

```diff
@@ -154,6 +154,8 @@ func makeReadRequest(deviceClient *opcua.Client, req sdkModel.CommandRequest) (*
 		return nil, fmt.Errorf("Driver.handleReadCommands: Status not OK: %v", resp.Results[0].Status)
 	}
 
+	// make new result
+	reading := resp.Results[0].Value.Value()
```

Comment:
What has this to do with a null pointer? And can the next line with `newResult(req, reading)` be compiled at all if the `reading := ...` is missing?